### ISSUE TITLE
Demo project doesn't work well

### ISSUE THREAD

**Issue by Piasy** [Danylo2006/LondonEyeLayoutManager#3]
When scroll down, it works well; but when scroll up, it's laggy, and when scroll up fast (fling), items will get out of order, or event collapsing.

**Comment by danylovolokh** [comment]
Yes, I am aware of that issue. It's because the scroll distance dy is bigger than the amount created points of the circle. The calculation of new location doesn't handle this case. Feel free to join the project and fix it :)
